doc/cards.xsd

```diff
@@ -29,12 +29,12 @@
                           <xs:extension base="xs:string">
                             <xs:attribute type="xs:int" name="muId" use="optional"/>
                             <xs:attribute type="xs:anyURI" name="picUrl" use="optional"/>
-                            <xs:attribute type="xs:anyURI" name="picUrlHq" use="optional"/>
```

Comment:
Why remove this? It's still valid, no? Just because we don't have a source right now doesn't mean someone might not want to supply one manually.